src/Backend/SgateShopgatePlugin/Helpers/WebCheckout.php

```diff
@@ -256,4 +258,16 @@ public function getSessionId()
 
         return session_id();
     }
+
+    public function closeRequest() {
```

Comment:
I like that approach much better :+1:  Hope they'll accept the NOSONAR annotation.